src/Microsoft.AspNet.Mvc.Core/Formatters/OutputFormatter.cs

```diff
@@ -201,11 +203,11 @@ public virtual void WriteResponseHeaders(OutputFormatterContext context)
             // Copy the media type as we don't want it to affect the next request
             selectedMediaType = selectedMediaType.Copy();
 
-            // Not text-based media types will use an encoding/charset - binary formats just ignore it. We want to
+            // Note text-based media types will use an encoding/charset - binary formats just ignore it. We want to
```

Comment:
If you're going to take the time to change it then it's fair game for feedback :trollface: